cmd/astrolabe/main.go

```diff
@@ -0,0 +1,312 @@
+package main
```

Comment:
would it be better to make the path to cmd/astrolabe_client/main.go to keep aligned with astrolabe_server naming?

Reply:
Sure

Reply:
I tried it, but if you change the name of the directory then it changes the name of the executable.  I'd prefer to have the executable named "astrolabe" since that is what people should be running, so I'm going to leave it as is.

Reply:
Ok. Then leave it as "astrolabe".